src/test/java/org/cloudfoundry/promregator/cache/AutoRefreshingCacheMapTest.java

```diff
@@ -133,6 +133,7 @@ public void run() {
 				Assert.assertEquals("labolg", value);
 			} catch (InterruptedException e) {
 				Assert.fail("Thread was interrupted");
+				Thread.currentThread().interrupt();
```

Comment:
Well, won't be reached, as `Assert.fail` never returns. 
But Sonar doesn't know this...

Shall I put a "false positive" to SonarCloud on these cases?

Reply:
Should we just take out the assert.fail and interrupt?

Reply:
Have put the Interrupt before the Assert, that way , we are still being good Thread Citizens by not dropping the interrupted status, and still allowing our assert to occur.

Reply:
That's a good compromise